### PR TITLE
Removing broken dapps

### PR DESCRIPTION
**Pull request by dievazqu**
### Description

Removing broken dapps that don't support wc v2.

Note: Disabling the dapps with the `listOn` flag for now, we can completely delete the dapps with their image and translations once we know they won't implement wc v2.

### CODE REVIEW

File: src/valora-dapp-list.json [hunk 1]
```diff
@@ -158,53 +158,53 @@
       "categoryId": "finance-tools",
       "categories": ["finance-tools", "earn"],
       "url": "https://www.immortaldao.finance",
-      "listOnAndroid": true,
-      "listOnIos": true
+      "listOnAndroid": false,
+      "listOnIos": false
```
Comment (ellenwoconnor) [inline src/valora-dapp-list.json:162]: mento did work for me, i think

File: src/valora-dapp-list.json [hunk 2]
```diff
@@ -320,17 +320,17 @@
       "categoryId": "exchanges",
       "categories": ["exchanges", "earn"],
       "url": "https://curve.fi/#/celo/swap",
-      "listOnAndroid": true,
-      "listOnIos": true
+      "listOnAndroid": false,
+      "listOnIos": false
```
Comment (ellenwoconnor) [inline src/valora-dapp-list.json:324]: this one might work bc i see some connections, let's leave as true for now

File: src/valora-dapp-list.json [hunk 3]
```diff
@@ -266,44 +266,44 @@
...
     },
     {
       "name": "Pixpress",
       "id": "pixpress",
       "categoryId": "exchanges",
       "categories": ["exchanges", "earn"],
       "url": "https://pixpress.pixelava.space",
-      "listOnAndroid": true,
-      "listOnIos": true
```
Comment (ellenwoconnor) [inline src/valora-dapp-list.json:279]: this one seems to work for me

File: src/valora-dapp-list.json [hunk 4]
```diff
@@ -266,44 +266,44 @@
       "categoryId": "lend-borrow-earn",
       "categories": ["earn"],
       "url": "https://app.stcelo.xyz/",
-      "listOnAndroid": true,
-      "listOnIos": true
+      "listOnAndroid": false,
+      "listOnIos": false
```
Comment (ellenwoconnor) [inline src/valora-dapp-list.json:270]: seems to be working as of today

File: src/valora-dapp-list.json (hunk 3 above)
Comment (dievazqu) [reply src/valora-dapp-list.json:279]: Oh, great, I'm reverting this then :)

File: src/valora-dapp-list.json (hunk 4 above)
Comment (dievazqu) [reply src/valora-dapp-list.json:270]: The page is not loading for me:
https://app.stcelo.xyz/

But I'm reverting it

**Review by MuckT — APPROVED** [review]

**Pull request merged**